internal/cli/server.go

```diff
@@ -121,7 +121,8 @@ func logRequestHandler(h http.Handler, logger logr.Logger) http.Handler {
 func logRequest(r *http.Request, log logr.Logger) {
 	uri := r.URL.String()
 	method := r.Method
-	log.V(1).Info("received request", "method", method, "uri", uri)
+	log.V(1).Info("received request",
```

Comment:
iirc the value here is not the absolute target TRACE_LEVEL but increases the current trace level. This results in logs appearing only when `TRACE_LEVEL` is set to `2`. This makes `TRACE_LEVEL=1` useless and boring since nothing happens there. I think we should even show the requests in `TRACE_LEVEL=0`, if not then at least in `TRACE_LEVEL=1`.

There is a problem with the liveness probe of the server pod though, which is set to the `api/v1/info` endpoint. This results in request logs like this appearing in regular intervals:

```
2021/08/24 09:54:42 epinio/5089 "level"=2 "msg"="received request"  "method"="GET" "uri"="/api/v1/info" "user"=""  
```

This is annoying and confusing to the user and we should find a way to filter it. Since that endpoint is useful in general, we don't want to completely filter it out in logs. We should better create a special endpoint for liveness/readiness probes that can be filtered out in logs. Then we can show requests in `TRACE_LEVEL=0` I think.

Feel free to do that here or we can create a separate issue for that.

Reply:
According [to the docs](https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#when-should-you-use-a-liveness-probe) we don't even need a liveness probe. Epinio server knows how to die when something goes wrong. 
We still need the readiness probe though.

Reply:
Will make a new endpoint, and then reduce level.

Reply:
Commits
- 8458f6aab0224a3d9d31f8754336e245a3d2fea5
- 67eee30fbb4dd49456df78acfee9d0ccbd444003

Reply:
Requests are now logged for TRACE_LEVEL=1 and higher.